### ISSUE TITLE
Change theme icon from hammer and screwdriver to T-shirt

### ISSUE THREAD

**Issue by Jermolene** [Jermolene/TiddlyWiki5#2264]
The T-shirt icon is [commonly use in Chinese smartphone apps](http://dangrover.com/blog/2015/01/31/more-chinese-mobile-ui-trends.html#some-icons-ive-only-seen-here) to represent theming functionality similar to that of TiddlyWiki.

![themetshirts 2x](https://cloud.githubusercontent.com/assets/174761/12727868/d7f6b456-c916-11e5-80ff-c505e3bd53d1.png)

Here's the current hammer and screwdriver icon:

<img width="122" alt="screenshot 2016-02-01 19 09 36" src="https://cloud.githubusercontent.com/assets/174761/12727963/660233e2-c917-11e5-812a-4e5d850d2251.png">

The major problem with the existing icon is that it suggests a low level system function without expressing any of the essence of the operation of changing the appearance of the app. The hammer suggests that it might break things. Even for users who have no experience of the icon, the T-shirt seems more inviting and distinctive, and the metaphor of changing the design of a T-shirt is quite good.

**Comment by twMat** [comment]
Haha! I like it and it's fun and adds yet another *friendly touch* to TW like Motovun Jack and the tiddler fish do. And definitely better than hammer etc. May I just suggest that it is as "clear" as possible then because I'd one might not get it at first otherwise. (That leftmost T-shirt for example.... I'd probably not understand if shown to me out of context.)

Spontaneously I'd come up with something like the following. I think people can guess what this is more easily and it's pretty. And even if they mistake it for a "palette" feature, it's rather close. But on the other hand it's more anonymous than a t-shirt. Or perhaps the opposite; maybe there's a political connotation?
![image](https://cloud.githubusercontent.com/assets/4534056/12733085/04168d3c-c939-11e5-9301-0d74703b2ae2.png)

**Comment by Jermolene** [comment]
> Haha! I like it and it's fun and adds yet another friendly touch to TW like Motovun Jack and the tiddler fish do. And definitely better than hammer etc. May I just suggest that it is as "clear" as possible then because I'd one might not get it at first otherwise. (That leftmost T-shirt for example.... I'd probably not understand if shown to me out of context.)

Yes, the proposal hinges on creating/finding acceptable artwork.

> Spontaneously I'd come up with something like the following. I think people can guess what this is more easily and it's pretty

I think that that's a candidate icon for "palette", which is of course a distinct concept from "themes" in TiddlyWiki. It's very similar to some experiments I tried when working on the current palette icon:

<img width="315" alt="screenshot 2016-02-01 22 35 58" src="https://cloud.githubusercontent.com/assets/174761/12733708/305330f8-c934-11e5-8fd2-e797d2bb7ade.png">

**Comment by telmiger** [comment]
Hmmm, let me share some thoughts about this. First of all, I agree: hammer and screwdriver symbolize manual work and are too technical, even if you think of them for building things. – What can a theme change actually?

- colours
- fonts
- shapes
- effects (transitions)

A T-shirt can have colours and it has a shape. It might even be available in different sizes. But I can’t see that in a single T-shirt. Maybe fonts and font sizes are more important? I looked through https://play.google.com/store/apps/category/PERSONALIZATION/collection/topselling_free  and https://play.google.com/store/search?q=desktop%20launcher&c=apps for inspiration – no T-shirts caught my eye here (seen from Europe). 
So maybe a simple shape and one or two letters could do the job: e.g. a square (with rounded corners?)  and Aa or Az or aZ ... or thin and bold a **a ** in it.

**Comment by tobibeer** [comment]
Perhaps that brush could be writing a colorful letter?

**Comment by telmiger** [comment]
I like the monochromatic Icons – maybe black and white, positive and
negative would be sufficient? Serif and sans serif fonts would be nice.
(Half letters?)
